cloud/pkg/cloudhub/common/model/types.go

```diff
@@ -170,5 +170,5 @@ func (event *Event) IsToEdge() bool {
 
 // GetContent dumps the content to string
 func (event *Event) GetContent() string {
-	return fmt.Sprintf("%v", event.Content)
+	return fmt.Sprintf("%s", event.Content)
```

Comment:
I think event.Content is of type interface{}, so we may need %v here?

Reply:
Yes, event.Content is of type interface{}.